### PR TITLE
Include user details in SSO User

### PR DESCRIPTION
**Pull request by WillTaylor22**
- [x] Changelog entry added.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
```
Comment (richtier) [inline CHANGELOG.md]: please use the format as per https://uktrade.atlassian.net/wiki/spaces/ED/pages/825786369/Changelog
(see "libraries" section)

File: directory_sso_api_client/models.py [hunk 2]
```diff
@@ -8,6 +8,9 @@
 class SSOUser(AbstractUser):
     session_id = models.TextField()
     hashed_uuid = models.CharField(max_length=200)
+    has_user_profile = models.BooleanField()
+    job_title = models.CharField(max_length=123)
```
Comment (richtier) [inline directory_sso_api_client/models.py]: to keep things simple we can use `models.TextField()`, which requires no `max_length`

File: tests/test_backends.py [hunk 3]
```diff
@@ -35,13 +35,27 @@ def test_remote_sso_backend_api_response_ok(
...
+            json={'id': 1,
+                  'email': 'jim@example.com',
+                  'hashed_uuid': 'thing',
+                  'user_profile': {
+                      'first_name': 'Jim',
+                      'last_name': 'Bloggs',
+                      'job_title': 'Dev',
+                      'mobile_phone_number': '555'
+                  }
+                  }
```
Comment (richtier) [inline tests/test_backends.py]: funky indentation :)

File: CHANGELOG.md (hunk 1 above)
Comment (WillTaylor22) [reply CHANGELOG.md]: Done

File: directory_sso_api_client/models.py (hunk 2 above)
Comment (WillTaylor22) [reply directory_sso_api_client/models.py]: Done

File: tests/test_backends.py (hunk 3 above)
Comment (WillTaylor22) [reply tests/test_backends.py]: Updated!

**Pull request merged**